packages/client/lib/protobuf/compiled.d.ts

```diff
@@ -1320,10 +1320,10 @@ export namespace market {
       eventId?: Uint8Array | null;
 
       /** NewKeyCard userWalletAddr */
-      userWalletAddr?: Uint8Array | null;
+      userWalletAddr: Uint8Array;
```

Comment:
hmm.. can you point me to why this is necessary? next time the schema changes this needs to be re-done.. so we either need a .patch file to re-apply or script this somehow into the `pb-gen` build step.

Reply:
right... this is to bypass the typescript error in [buildState](https://github.com/masslbs/Tennessine/pull/27/files#diff-e0ca6374c72417e902d431ff5105811ac53e22b3b601a105ab94f54c390ddf17R224).

it doesn't like that we are trying to call bytesToHex on a value thats possibly undefined

Reply:
Hmm.. can you point me to the line where this is a problem? I think you should be able to work past that by using `event.field!` when passing the value but I'm not sure.

Reply:
https://github.com/masslbs/Tennessine/pull/27/files#diff-e0ca6374c72417e902d431ff5105811ac53e22b3b601a105ab94f54c390ddf17R224

okay yes i will do the bang instead